marblerun/workflows/define-manifest.md

```diff
@@ -156,10 +156,10 @@ When defining a custom key or certificate, the following fields are available.
 
 * `Type`: can be either `symmetric-key` for a symmetric encryption key, `cert-rsa`, `cert-ecdsa`, `cert-ed25519` or `plain`. Secrets of type `plain` contain arbitrary data uploaded by users, and are never generated by the Coordinator.
 * `Size`: the size of the key in bits. For symmetric keys, this needs to be a multiple of `8`. For ECDSA, this needs to map to a curve supported by Go's crypto library, currently: `224`, `256`, `384`, or `521`. For Ed25519, this should be ommitted.
-* `Shared` (default: `false`): specifies if the secret should be shared across all Marbles (`true`), or if the secret should be uniquely generated for each Marble (`false`). See [Secrets management](content/features/secrets-management.md) for more info.
+* `Shared` (default: `false`): specifies if the secret should be shared across all Marbles (`true`), or if the secret should be uniquely generated for each Marble (`false`). See [Secrets management](features/secrets-management.md) for more info.
 * `ValidFor` (only for certificates, default: `365`): validity of the certificate in days; cannot be specified in combination with the `NotAfter`.
 * `Cert` (only for certificates): allows for the specification of additional X.509 certificate properties. See below for details.
-* `UserDefined` (default: `false`): specifies if the secret should be generated by MarbleRun  (`false`), or if it will be uploaded by a user at a later date  (`true`).
+* `UserDefined` (default: `false`): specifies if the secret should be generated by MarbleRun  (`false`), or if it will be [uploaded by a user](workflows/managing-secrets.md) at a later date  (`true`).
```

Comment:
```suggestion
* `UserDefined` (default: `false`): specifies if the secret should be generated by MarbleRun (`false`), or if it will be [uploaded by a user](workflows/managing-secrets.md) at a later date  (`true`).
```